service/lib/ops/policyOps.js

```diff
@@ -22,12 +23,16 @@ function toArray (policies) {
 }
 
 function insertPolicies (client, policies, cb) {
-  policies = toArray(policies)
+  policies = toArray(policies).map((policy) => {
+    policy.id = policy.id || uuidV4()
 
-  const sql = SQL`INSERT INTO policies (version, name, org_id, statements) VALUES `
-  sql.append(SQL`(${policies[0].version}, ${policies[0].name}, ${policies[0].org_id}, ${policies[0].statements})`)
+    return policy
+  })
```

Comment:
I don't like this map here, because we are already looping through policies in `toArray` and then when creating the query.

Is it feasible to move the id initialisation into toArray (preferred) or into the query creation?